### PR TITLE
(DOCSP-26280) [Admin API] Deployment Config

### PR DESCRIPTION
**Pull request by nlarew**
### Jira

- (DOCSP-26280) [Admin API] Deployment Config

### Staged Changes (Requires MongoDB Corp SSO)

- [Admin API > Deployment](https://docs-atlas-staging.mongodb.com/atlas-app-services/docsworker-xlarge/admin-deployment/admin/api/v3/#tag/deploy/operation/adminGetDeploymentConfig)

### Review Guidelines

[REVIEWING.md](https://github.com/mongodb/docs-app-services/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/openapi-admin-v3.yaml [hunk 1]
```diff
@@ -2618,6 +2618,40 @@ paths:
...
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/DeploymentConfig"
+    patch:
+      tags:
+        - deploy
+      operationId: adminConfigureDeployment
+      summary: Configure Deployment
+      description: Modify an App's deployment configuration to customize how the app deploys changes.
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:2644]:
```suggestion
      description: Modify an App's deployment configuration to customize how the App deploys changes.
```

File: source/openapi-admin-v3.yaml [hunk 2]
```diff
@@ -2618,6 +2618,40 @@ paths:
...
+  "/groups/{groupId}/apps/{appId}/deploy/config":
+    parameters:
+      - "$ref": "#/components/parameters/GroupId"
+      - "$ref": "#/components/parameters/AppId"
+    get:
+      tags:
+        - deploy
+      operationId: adminGetDeploymentConfig
+      summary: Get the Deployment Configuration
+      description: See information about how your app deploys changes.
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:2631]:
```suggestion
      description: See information about how your App deploys changes.
```

File: source/openapi-admin-v3.yaml [hunk 3]
```diff
@@ -4266,6 +4300,39 @@ components:
...
+              type: array
+              description: |-
+                The unique ID values of automatic deployment provider
+                installations. For example, the ID of a GitHub app
+                installation.
+              items:
+                type: string
+        last_modified:
+          type: number
+          description: The time this configuration was last modified represented by the number of seconds since the UNIX epoch.
```
Comment (mongodben) [inline source/openapi-admin-v3.yaml:4335]: what if anything is required here?

**Review by mongodben — COMMENTED** [review]
back to you w Q

File: source/openapi-admin-v3.yaml (hunk 3 above)
Comment (nlarew) [reply source/openapi-admin-v3.yaml]: Updated w/ required fields

**Review by mongodben — APPROVED** [review]

**Pull request merged**